docs/version0.18.md

```diff
@@ -99,6 +99,13 @@ Support for Concurrent scavenge mode is now extended to macOS. For more informat
 
 The previously undocumented option [`-Xgc:noConcurrentScavenge`](xgc.md#noconcurrentscavenge) disables pause-less garbage collection.
 
+### Support for OpenJDK HotSpot options
```

Comment:
Please add this to the list at the top of the topic as per other items in the release notes.

Reply:
My apologies, didn't see the topics indices. 
See change here:
https://github.com/eclipse/openj9-docs/pull/464/commits/27ad960b4d2a8bb5eb0a9435ed38c03d1c7fc5ca